docroot/modules/custom/va_gov_facilities/src/EventSubscriber/FacilitiesSubscriber.php

```diff
@@ -112,13 +126,58 @@ public static function getSubscribedEvents(): array {
       'hook_event_dispatcher.form_node_vet_center_form.alter' => 'alterVetCenterNodeForm',
       'hook_event_dispatcher.form_node_vba_facility_service_edit_form.alter' => 'alterVbaFacilityServiceNodeForm',
       'hook_event_dispatcher.form_node_vba_facility_service_form.alter' => 'alterVbaFacilityServiceNodeForm',
+      'hook_event_dispatcher.form_node_vet_center_facility_health_servi_edit_form.alter' => 'alterVetCenterServiceNodeForm',
       EntityHookEvents::ENTITY_PRE_SAVE => 'entityPresave',
       EntityHookEvents::ENTITY_UPDATE => 'entityUpdate',
+      EntityHookEvents::ENTITY_VIEW_ALTER => 'entityViewAlter',
       FieldHookEvents::WIDGET_COMPLETE_FORM_ALTER => 'widgetCompleteFormAlter',
       FormHookEvents::FORM_ALTER => 'formAlter',
     ];
   }
 
+  /**
+   * Alteration to entity view pages.
+   *
+   * @param \Drupal\core_event_dispatcher\Event\Entity\EntityViewAlterEvent $event
+   *   The entity view alter service.
+   */
+  public function entityViewAlter(EntityViewAlterEvent $event):void {
+    $this->appendServiceDescriptionToVetCenterFacilityService($event);
+  }
+
+  /**
+   * Appends Vet Center facility service description to it's title on node:view.
```

Comment:
```suggestion
   * Appends Vet Center facility service description to its title on node:view.
```
SHOULD: The possessive pronoun doesn't have an apostrophe.